src/sykmeldt/components/oppfolgingsplan/godkjentplan/GodkjentPlan.tsx

```diff
@@ -0,0 +1,74 @@
+import { StatusHeader } from "@/common/components/wrappers/old/StatusHeader";
+import { Row } from "@/common/components/wrappers/Row";
+import { SpacedDiv } from "@/common/components/wrappers/SpacedDiv";
+import { HakeGronnLysImage } from "@/common/images/imageComponents";
+import { Oppfolgingsplan } from "../../../../schema/oppfolgingsplanSchema";
+import { AvbrytPlanKnapp } from "../godkjenn/AvbrytPlanKnapp";
+import { SePlan } from "../godkjenn/SePlan";
+import { TidligereOppfolgingsplaner } from "../godkjenn/TidligereOppfolgingsplaner";
+import { TilbakeLenke } from "../godkjenn/TilbakeLenke";
+import { BothApprovedOppfolgingsplan } from "./BothApprovedOppfolgingsplan";
+import { DelMedFastlegeKnapp } from "./DelMedFastlegeKnapp";
+import { DelMedNavKnapp } from "./DelMedNavKnapp";
+import { ForcedApprovedOppfolgingsplan } from "./ForcedApprovedOppfolgingsplan";
+import { GodkjentPlanDeltBekreftelse } from "./GodkjentPlanDeltBekreftelse";
+import { HvaSkjerNa } from "./HvaSkjerNa";
+import { LastNed } from "./LastNed";
+
+interface Props {
+  oppfolgingsplan: Oppfolgingsplan;
+}
+
+export const GodkjentPlan = ({ oppfolgingsplan }: Props) => {
+  if (
+    !oppfolgingsplan.godkjentPlan ||
+    !oppfolgingsplan.arbeidsgiver?.naermesteLeder
+  ) {
+    return null;
+  }
+
+  const godkjentPlan = oppfolgingsplan.godkjentPlan;
+
+  return (
+    <SpacedDiv>
+      <StatusHeader svgUrl={HakeGronnLysImage} tittel={"Oppfølgingsplanen"} />
```

Comment:
En ting som ble litt teit er dobbel header. Får heading fra <Side> komponenten. Må kanskje velge 1, og sørge for at den er H1